Dockerfile

```diff
@@ -152,7 +153,8 @@ RUN mkdir $HOME/sim/build \
         && cd circuitBuilding_1000neurons \
         && sed -i "s/CircuitTarget.*/CircuitTarget mini50/g" BlueConfig \
         && sed -i "s/RunMode.*/RunMode RR/g" BlueConfig \
-        && sed -i "s/^Report /#Report /g" BlueConfig
+        && sed -i "s/^Report /#Report /g" BlueConfig \
+	&& cp BlueConfig ../../circuitBuilding_1000neurons
```

Comment:
why we need to copy BlueConfig back? If I am not maitaken we run simulation in the current directory.

Reply:
Yes, you are totally right. I misread the readme and thought you needed to run the simulation on sim/circuitBuilding, but you do it on sim/build/circuitBuilding that is where the BlueConfig is located...